build.sbt

```diff
@@ -68,14 +76,18 @@ lazy val anorm = project
       // New functions
       missMeth("anorm.Sql.unsafeStatement$default$2"),
       missMeth("anorm.Sql.unsafeResultSet"),
-      missMeth("anorm.Sql.unsafeStatement")),
+      missMeth("anorm.Sql.unsafeStatement"),
+      missMeth("anorm.ToStatement.contramap"),
```

Comment:
In general with Play, our policy is to avoid breaking the ABI in minor releases, except to fix serious bugs. Slick has a similar policy. It's extremely common for external modules to not upgrade their Play version between releases. So this either causes breakage, or prevents people from upgrading until they can get all modules recompiled for the new version.

Hard to say whether anorm will have the same problems, but if it has any significant number of users it likely will.

Often, we'll have two different implementations of the same change, one that breaks binary compatibility in the future release branch (currently 2.6.x) and another that preserves binary compatibility on the stable release branch (2.5.x). To me this is a good compromise: we get to eventually have the code the way we want it, but preserve binary compatibility for existing users.